examples/testerina-mocking-functions/testerina_function_mocking.description

```diff
@@ -0,0 +1,6 @@
+// Mock functions allow you to hide the real functions and engage your own functions when running tests.
+// This allows you to isolate your test functions from the rest. Both mocking an imported function and a
+// function declared in the module under test can be mocked with dedicated annotations.<br/><br/>
+// Mocking an imported function will apply the mocked function to every instance of the original function call.
+// It is not limited to the test file where it is being mocked. Mocking a function from the module under test
+// can be scoped to have different behaviors for different test cases.
```

Comment:
We need to add a line pointing to the API docs as well. 

```
/ For more information, see [Testing Ballerina Code](https://ballerina.io/learn/testing-ballerina-code/).
```

Reply:
Added